src/mixins/borrow/cooks.js

```diff
@@ -1242,10 +1244,14 @@ export default {
       }
 
       // 30 unwrap and deposit for alice in degenbox
+      await this.createMimBentoInfo();
```

Comment:
No idea if this is ok to do... But need a way to get access to the Bento contract.